web-basics-browser-security-model/activities/my-special-name/sol/solution

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+PORT=8088
+
+if [[ $1 == "local" ]]
+then
+    remote='http://127.0.0.1:'$PORT
+elif [[ $1 == "remote" ]] && [[ -z $2 ]] 
+then
+    remote='http://141.85.224.157:'$PORT
+else
+    remote=$1':'$2
+fi
+
+# Gimme
```

Comment:
This challenge is My Special Name, not Gimme

Reply:
Yeah, it seems that I didn't include the right script for my special name.
It should be ok now.